### DISCUSSION TITLE
Updating the storageclass type post deployment

### DISCUSSION THREAD

**Discussion by saloni-2404** [strimzi/strimzi-kafka-operator#4282] [General]
Hi,
Is it possible to update the storage class type of pvc of an existing kafka deployment?

**Comment by scholzj** [comment]
No, that is not possible. The storage class is not updatable => it often represents different type of storage and you cannot justlike that turn spinning hard-drive into SSD or move it to another SAN. So it would in many cases mean that you have to create new disk, copy data etc.

If you use the JBOD storage, you can add new volume with new storage class, move the partitions to the new volume and remove the old one.

**Reply by saloni-2404** [reply]
Okay thanks Jakub!
